server/s3_object_action_base.h

```diff
@@ -48,17 +48,28 @@ class S3ObjectAction : public S3Action {
  protected:
   std::shared_ptr<S3ObjectMetadata> object_metadata;
   std::shared_ptr<S3BucketMetadata> bucket_metadata;
+  std::shared_ptr<S3BucketMetadata> additional_bucket_metadata;
```

Comment:
Have comment about the usage of additional_bucket_metadata additional_object_metadata

Reply:
done